### PR TITLE
Fix Migration Imports

### LINKED ISSUE #4299: Migration: Turn tests back on after NuGet imports goes in

### LINKED ISSUE #4339: Migration: Imports not migrating properly

### PR DESCRIPTION
**Pull request by brthor**
Blocked on https://github.com/NuGet/Home/issues/3621

Fixes #4339 
Fixes #4299 

@eerhardt @livarcocc

### CODE REVIEW

**Comment by livarcocc** [comment]
:shipit: when green.

File: src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigratePackageDependenciesAndToolsRule.cs [hunk 1]
```diff
@@ -69,17 +69,21 @@ public void Apply(MigrationSettings migrationSettings, MigrationRuleInputs migra
...
+
             if (transform != null)
             {
                 transform.Condition = targetFramework.FrameworkName.GetMSBuildCondition();
-                _transformApplicator.Execute(transform, commonItemGroup);
+                _transformApplicator.Execute(transform, commonPropertyGroup);
+            }
+            else
+            {
+                MigrationTrace.Instance.WriteLine($"{nameof(MigratePackageDependenciesAndToolsRule)}: imports transform null for {string.Join(";", targetFramework.FrameworkName.GetShortFolderName())}");
```
Comment (eerhardt) [inline src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigratePackageDependenciesAndToolsRule.cs]: Why the `string.Join` here?  Isn't `targetFramework.FrameworkName.GetShortFolderName()` a single string?

**Comment by eerhardt** [comment]
:shipit:

**Comment by brthor** [comment]
test this

**Comment by brthor** [comment]
test this please

**Comment by brthor** [comment]
test this please

**Comment by brthor** [comment]
@dotnet-bot test this please

**Pull request merged**